app/controllers/catalog_controller.rb

```diff
@@ -204,8 +204,7 @@ class CatalogController < ApplicationController
     # label in pulldown is followed by the name of the SOLR field to sort by and
     # whether the sort is ascending or descending (it must be asc or desc
     # except in the relevancy case).
-    config.add_sort_field 'score desc, title_sort asc', label: 'relevance'
```

Comment:
Yes, the default behavior here is what we'd like. But I think people would still like to have a relevance sort — can you just flip the order instead of removing relevance?